omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/RelationshipTypeController1_8Test.java

```diff
@@ -102,6 +103,25 @@ public void shouldRetireARelationshipType() throws Exception {
 		Assert.assertEquals("test reason", relationshipType.getRetireReason());
 	}
 	
+	@Test
+	public void shouldUnRetireARelationshipType() throws Exception {
+		RelationshipType relationshipType = service.getRelationshipTypeByUuid(RestTestConstants1_8.RELATIONSHIP_TYPE_UUID);
+		relationshipType.setRetired(true);
+		relationshipType.setRetiredBy(Context.getAuthenticatedUser());
+		relationshipType.setDateRetired(new Date());
+		relationshipType.setRetireReason("random reason");
+		service.saveRelationshipType(relationshipType);
```

Comment:
What happens if you comment out the above line?

Reply:
This has been resolved.